src/couch/src/couch_bt_engine_header.erl

```diff
@@ -229,6 +236,7 @@ upgrade_disk_version(#db_header{}=Header) ->
         3 -> throw({database_disk_version_error, ?OLD_DISK_VERSION_ERROR});
         4 -> Header#db_header{security_ptr = nil}; % [0.10 - 0.11)
         5 -> Header; % pre 1.2
+        6 -> Header;
```

Comment:
@davisp on reflection, I don't think I need to bump disk version as we're only adding a new field.